content/blog/2019/08/2019-08-17-role-strategy-performance-folder-auth.adoc

```diff
@@ -0,0 +1,100 @@
+
+All performance improvements were released in Role Strategy Plugin
+version link:https://github.com/jenkinsci/role-strategy-plugin/releases/tag/role-strategy-2.13[2.13].
+
+= The Folder Auth plugin
+
+The performance improvements don't stop there. We created the brand new
+link:https://github.com/jenkinsci/folder-auth-plugin[Folder Authorization] plugin
+from the ground-up to make things even faster. This new plugin is designed to
+improve the performance for projects organized in folders from the
+plugin:cloudbees-folder[Cloudbees' Folders] plugin without having to break any
+backward compatibility.
+
+image::/images/post-images/role-strategy-performance/folder-auth.png[Screenshot of the Folder Auth Plugin]
+
+Just like the Role Strategy Plugin, there are three types of roles supported:
+
+* Global Roles: applicable everywhere in Jenkins
+* Agent Roles: restricting permissions to multiple agents connected to your instance
+* Folder Roles: applicable to multiple jobs organized inside folders
+
+To make the task of managing roles simpler and improve the performance, the
+plugin does not use regular expressions. Permissions given to a folder through
+a folder role get inherited to all of its children. This is useful for giving
+access to multiple projects through a single role. Similarly, an agent role
+can be applied to multiple agents and assigned to multiple users.
+
+Benchmarks for identical configurations of this plugin show that the
+permissions check are up to 934x faster for 500 global roles when compared to
+the global roles from the Role Strategy 2.13, containing all performance
+improvements. Folder roles are faster too. For similar configurations using
+regular expressions in the Role Strategy Plugin, a permission check for access
+to a job almost 15x faster.
+
+The plugin supports Jenkins Configuration-as-Code code and provides REST APIs
+for managing roles with Swagger API support. Checkout the documentation on 
+link:https://github.com/jenkinsci/folder-auth-plugin/blob/master/README.md[GitHub].
+The 1.0 version of the plugin has just been and be released and can be downloaded
+from your Jenkins' Update center.
+
+== Links and Feedback
+I would love to hear your comments and suggestions. Please feel free to reach
+out to me through either the
+link:https://gitter.im/jenkinsci/role-strategy-plugin[Role Strategy Plugin Gitter chat] or through
+link:mailto:jenkinsci-dev@googlegroups.com[Jenkins Developer Mailing list].
+
+* link:https://drive.google.com/file/d/1IVe3T8WdTILmb62PAIJveR4KbBWzPt1k/view?usp=sharing[Presentation slides for second phase evaluations]
+* link:https://youtu.be/tnoObQqGhyM?t=1101[Demo for the alpha version of Folder Auth]
+* link:https://github.com/jenkinsci/folder-auth-plugin/blob/master/README.md[Documentation for the Folder Auth Plugin]
+* link:https://github.com/jenkinsci/role-strategy-plugin/releases/tag/role-strategy-2.13[Release of the Role Strategy Plugin with performance improvements]
```

Comment:
I would remove a link to the specific release. There were performance improvements before it, and I am sure there will be more changes later